src/components/Notespage/InputBox.jsx

```diff
@@ -41,15 +67,52 @@ export const InputBox = () => {
 							Save Note
 						</button>
 						<div className="selective-cta">
-							<span>
+							<span
+								onClick={() =>
+									colorBox ? setcolorBox(false) : setcolorBox(true)
+								}
+								className="note-cta"
+							>
 								<i className="far fa-palette"></i>
 							</span>
-							<span>
+							<span
+								className="note-cta"
+								onClick={() =>
+									labelBox ? setLabelBox(false) : setLabelBox(true)
+								}
+							>
 								<i className="far fa-tag"></i>
 							</span>
-							<label>label</label>
+							<label>{label}</label>
 						</div>
 					</div>
+					<div
+						style={colorBox ? { display: "flex" } : { display: "none" }}
+						className="color-div"
+					>
+						{colourPalette.map((color) => (
```

Comment:
A suggestion : You could add an option to clear a selected color, or just come out of the color option.
For example. 
I write a note, I think I want to color it. So I open color options. I realize I dont like any colors. 
I dont have an option to just close the menu. So a small cross button, or an option in the end which basically clears the colors , might do better for the overall UX of the project I think.

Reply:
sure , thats a good idea , i will implement that